lib/cc/yaml/nodes/sequence.rb

```diff
@@ -113,10 +113,10 @@ def add_value!(value)
 
       protected
 
-        def dup_values
-          @children = @children.map { |child| child.dup }
-          self
-        end
+      def dup_values
+        @children = @children.map { |child| child.dup }
+        self
+      end
```

Comment:
I would revert this just to keep the diff cleaner. Our rule is not to change code *just* to change style.